docs/start/authentication/graphql-storefront.mdx

```diff
@@ -119,6 +123,18 @@ Unlike storefront API tokens, customer impersonation tokens _are_ sensitive and
 
 If your token is compromised, you can use the [Revoke a token](/docs/rest-authentication/tokens#revoke-a-token) endpoint. Only use this in emergencies; do not revoke tokens unnecessarily. Instead, use a shorter expiration and allow them to expire naturally.
 
+## Development mode
```

Comment:
Thank you for adding section. Could we remove the references to the dev mode here https://github.com/bigcommerce/docs/pull/379/files#diff-fe58c824ab743b6dd3d3b352d178a7884f5cc49b28548c7d06b92a37627f1876R102 and here https://github.com/bigcommerce/docs/pull/379/files#diff-fe58c824ab743b6dd3d3b352d178a7884f5cc49b28548c7d06b92a37627f1876R53

Reply:
K they are removed now, I must have missed your comment earlier